### PR TITLE
Code/ocr countries list

### PR DESCRIPTION
**Pull request by Frank-Howd**
Add efficiency to code base, ocr.py; removed redundant code through use of sets to generate countries list and remove non-matches

### CODE REVIEW

File: app/ocr.py [hunk 1]
```diff
@@ -287,20 +289,24 @@ def get_country_of_origin(self):
...
-        if "Puerto Rico" in countries:
-            countries.remove("Puerto Rico")
-        if "United States" in countries:
-            countries.remove("United States")
-        if "United States Minor Outlying Islands" in countries:
-            countries.remove("United States Minor Outlying Islands")
-        if "U.S. Virgin Islands" in countries:
-            countries.remove("U.S. Virgin Islands")
+        countries = set(countries)
+        non_matches = {"American Samoa", "Guam", "Northern Mariana Islands", "Puerto Rico",
```
Comment (Dylan-Sivori) [inline app/ocr.py:291]: Very smart to use a set here!

File: app/ocr.py [hunk 2]
```diff
@@ -19,7 +19,9 @@
 from spacy.matcher import Matcher, PhraseMatcher
 from typing import List, Tuple, Union, Callable, Dict, Iterator
 
-nlp = load("en_core_web_md")
+# change to small
+# nlp = load("en_core_web_md")
```
Comment (Dylan-Sivori) [inline app/ocr.py]: Thank you for documenting this change. Could you remove the commented our code so this is not left in the codebase?

**Review by Dylan-Sivori — COMMENTED** [review]
Great idea to improve the code by using a set here. It looks like there are a few comments that were leftover. Also it seems that we have a merge conflict with this pull request. Once we get that resolved it should be able to merge. Great job!

File: app/ocr.py (hunk 2 above)
Comment (Frank-Howd) [reply app/ocr.py]: Good eyes.  Will do, Dylan!

Comment (Dylan-Sivori) [reply app/ocr.py]: Thanks Frank! I believe this is also where we are getting that merge conflict. The "from typing" import and loading the "en_core_web_sm" from spacy are conflicting with main. Everything looks good, just have to resolve those conflicts before this branch is able to merge.

**Review by Dylan-Sivori — APPROVED** [review]

**Comment by Frank-Howd** [comment]
Will close this branch, and rework from a new pull/new branch.

**Pull request closed without merging**